requirements.txt

```diff
@@ -853,9 +853,9 @@ typing-extensions==4.8.0 \
     #   black
     #   pydantic
     #   pydantic-core
-urllib3==1.26.16 \
-    --hash=sha256:8d36afa7616d8ab714608411b4a3b13e58f463aee519024578e062e141dce20f \
-    --hash=sha256:8f135f6502756bde6b2a9b28989df5fbe87c9970cecaa69041edcce7f0589b14
+urllib3==1.26.18 \
```

Comment:
I assume this is intentional in this PR?

Reply:
That change is just from my merging the latest `main` into this PR. If you look at the full diff view (ie not just the latest commits) you can see there isn't any change to `urllib3`.